haproxy/files/haproxy.cfg

```diff
@@ -14,10 +14,6 @@ global
   tune.maxrewrite 1024
   tune.bufsize 32768
   maxconn  16000
-  ca-base /etc/ssl/certs
-  crt-base /etc/ssl/private
-  ssl-default-bind-ciphers ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5:!DSS
```

Comment:
How is it ensured that insecure SSLv3 or weak ciphers won't be used? This should be at least parametrized with some sane default (same as eg. in apache formula).